examples/dft/run_time_dispatching/real_fwd_usm.cpp

```diff
@@ -42,33 +47,34 @@ void run_uniform_example(const sycl::device& dev) {
         std::exit(2);
     };
 
-    sycl::queue queue(dev, exception_handler);
-
     std::cout << "DFTI example run_time dispatch" << std::endl;
-    //
-    // Preparation on cpu
-    //
-    sycl::queue cpu_queue(dev, exception_handler);
-    sycl::context cpu_context = cpu_queue.get_context();
-    sycl::event cpu_getrf_done;
 
-    double *x_usm = (double*) malloc_shared(N*2*sizeof(double), cpu_queue.get_device(), cpu_queue.get_context());
+    sycl::queue sycl_queue(dev, exception_handler);
+    auto x_usm = sycl::malloc_shared<float>(N * 2, sycl_queue);
 
-    // enabling
-    // 1. create descriptors 
-    oneapi::mkl::dft::descriptor<oneapi::mkl::dft::precision::DOUBLE, oneapi::mkl::dft::domain::COMPLEX> desc_vector({N,N});
+    // 1. create descriptors
+    oneapi::mkl::dft::descriptor<oneapi::mkl::dft::precision::SINGLE,
+                                 oneapi::mkl::dft::domain::REAL>
+        desc(N);
 
     // 2. variadic set_value
-    desc_vector.set_value(oneapi::mkl::dft::config_param::BACKWARD_SCALE, (double)(1.0/N));
-    desc_vector.set_value(oneapi::mkl::dft::config_param::NUMBER_OF_TRANSFORMS, 4);
-    desc_vector.set_value(oneapi::mkl::dft::config_param::BWD_DISTANCE, N);
-    desc_vector.set_value(oneapi::mkl::dft::config_param::PLACEMENT, oneapi::mkl::dft::config_value::NOT_INPLACE);
+    desc.set_value(oneapi::mkl::dft::config_param::FORWARD_SCALE, 1.f / N);
+    desc.set_value(oneapi::mkl::dft::config_param::NUMBER_OF_TRANSFORMS,
+                   static_cast<std::int64_t>(1));
+    desc.set_value(oneapi::mkl::dft::config_param::PLACEMENT,
+                   oneapi::mkl::dft::config_value::INPLACE);
+
+    // 4. commit_descriptor (runtime xPU)
```

Comment:
There's no step 3 there. `xPU`?

Reply:
Renumbered steps in [95f4e2c](https://github.com/hjabird/oneMKL/pull/6/commits/95f4e2ce326ac7a1c78068056ff9bbf3a400d705). I've replaced *runtime xPU* with *runtime dispatch*.